docs/source/05-01-building-basic-plugin.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Creating a Basic Pride Reaction Plugin
+permalink: /building-basic-plugin/
+---
+
+In this tutorial, we will build a basic reaction plugin.
+
+## What is a plugin?
+
+Talk has two parts - the first is core. Our core code includes all commenting and moderation features that are necessary for a comment section, and ones that we believe are important to be universal. This code can be found in our [Talk repo](https://github.com/coralproject/talk).
+
+The other part is plugins. Plugins are additional functionality which are optional to use with Talk. You can turn these on or off, depending on your specific needs. Plugins are either part of our core plugins, which ship with Talk, or they are developed by 3rd parties and either used privately and internally, or are open sourced for use across the greater community.  
+
+## Reactions
+
+Talk exposes a friendly API to create new reactions. To explore the capabilities of Talk we are going to create a new reaction together step-by-step.
+
+In Talk, there are currently three ways commenters can react to comments: Like, Love, and Respect. These reactions are separated into plugins so that you can customize which reactions you want to use by toggling them on or off - or by adding your own custom reactions, which is what we are going to do today.
+
+We can create a new plugin from scratch or we can use the Talk CLI to generate a plugin template for us to use. CLI stands for Command Line Interface, meaning can access utilities via the command line that make it easy to interact and integrate with Talk.
+
+Please note this tutorial assumes you have already [installed and configured Talk locally](https://docs.coralproject.net).
```

Comment:
Should reference the specific page in the docs using a relative link:

```diff
-Please note this tutorial assumes you have already [installed and configured Talk locally](https://docs.coralproject.net).
+Please note this tutorial assumes you have already [installed and configured Talk locally](/talk/)
```

Reply:
Fixed!